docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
Demos I have kept as weak as possible.  The only change was I had to change `extends React.Component` to `extends React.Component<any, any>`